web/styles/app_variables.css

```diff
@@ -559,6 +559,8 @@
     /* Shrink the user activity circle for the Recent Conversations context. */
     /* 7px at 14px/1em */
     --length-user-status-circle-recent-conversations: 0.5em;
+    --length-user-status-circle-popover-menu: 8px;
+    --length-user-popover-menu: 64px;
```

Comment:
These variable names are confusing; `length-user-popover-menu` should have `avatar` in the name, since that's the size of the avatar.

Reply:
Thank you! That's much better